syntax = "proto2";

package dharma.canbus;

import "modules/common/proto/header.proto";

message Chassis {

  // chassis also needs it own sending timestamp
  optional dharma.common.Header header = 1;
  // Vehicle Speed in meters per second.
  optional double speed_mps = 2 [default = nan];

}
